types/mempool/mempool_test.go

```diff
@@ -232,19 +232,11 @@ func (s *MempoolTestSuite) TestSampleTxs() {
 	require.NoError(t, err)
 	require.NoError(t, mp.Insert(ctxt, delegatorTx))
 	require.Equal(t, 1, mp.CountTx())
-
-	proposalTx, err := unmarshalTx(msgMultiSigMsgSubmitProposal)
```

Comment:
I think we should keep this test, but use another gov proposal. Otherwise we test less.

Reply:
we could use another gov proposal, but the thing is the msg we are using should actually implement gov v1beta1.Content interface. That would introduce module dependency in gov right?